cmd/dockerd/daemon.go

```diff
@@ -354,13 +354,19 @@ func (cli *daemonCLI) start(ctx context.Context) (err error) {
 		log.G(ctx).WithError(err).Error("Failed to close buildkit")
 	}
 
+	log.G(ctx).Info("closed buildkit")
```

Comment:
Accidentally added an irrelevant commit?

Reply:
Not accidental - this PR's still draft, and that's part of temporary commit [TEMP: dump daemon logs after d.Stop (debugging error-exit)](https://github.com/moby/moby/pull/48593/commits/205569bee1a2ea0642ba76acc445d824954e6482) ...

For some reason, the daemon's not shutting down cleanly in CI (it's fine in a dev env). I think it may be something to do with OTEL tracing, because with that extra debug the daemon log ends with ...

```
        time="2024-10-08T09:26:07.239164944Z" level=debug msg="Clean shutdown succeeded"
        time="2024-10-08T09:26:07.239249993Z" level=info msg="closed buildkit"
        time="2024-10-08T09:26:07.239258238Z" level=info msg=cancelled
        time="2024-10-08T09:26:07.239262526Z" level=info msg="errAPI !ok"
        time="2024-10-08T09:26:27.238043935Z" level=info msg="Processing signal 'interrupt'"
        time="2024-10-08T09:26:28.237981737Z" level=info msg="Processing signal 'interrupt'"
        time="2024-10-08T09:26:29.238348730Z" level=info msg="Processing signal 'interrupt'"
        time="2024-10-08T09:26:29.238384185Z" level=info msg="Forcing docker daemon shutdown without cleanup; 3 interrupts received"
```

So, it looks like it didn't come back from [tp.Shutdown](https://github.com/moby/moby/blob/205569bee1a2ea0642ba76acc445d824954e6482/cmd/dockerd/daemon.go#L370).), but I'm still investigating ...

Reply:
There's no routing set up in the netns where the daemon's running during this test, so the OTEL endpoint isn't reachable and that misconfiguration causes the daemon to get stuck during exit ... not sure it should, but that's a separate discussion.

For now, I've disabled OTEL for this test, and got rid of that temporary commit.